06-management/03-migration-and-backup.md

```diff
@@ -0,0 +1,110 @@
+
+You can skip the step of exporting schema if you already have a copy of your schema to import.
+
+```
+old/grakn server schema [keyspace] > schema.gql
+new/grakn console -k [keyspace] -f schema.gql
+```
+
+### Data Migration
+
+Data migration is performed using an export followed by an import.
+
+```
+old/grakn server export [keyspace] data.grakn
+new/grakn server import [keyspace] data.grakn
+```
+
+This requires your keyspace in the new grakn to have a valid schema that is compatible with your data. If a failure occurs during import, please check your keyspace has the schema you expect.
+
+Once the data has been successfully imported, you can safely delete the temporary data file with `rm data.grakn`.
+
+## Upgrading Grakn with Homebrew
+
+When upgrading a Grakn installation via homebrew, we recommend that you always back up your grakn directory first to preserve your data in case the upgrade cannot be completed successfully. This is because homebrew automatically removes your installation directly after the upgrade.
+
+`cp -R /usr/local/Cellar/grakn-core/[version]/libexec ./grakn-backup`.
+
+Before upgrading, your should first export your schema and data. You should perform this step for every keyspace you want to upgrade.
+
+```
+grakn server schema [keyspace] > [keyspace].gql
+grakn server export [keyspace] [keyspace].grakn
+```
+
+## Dealing With Migration Issues
+
+### Migration Errors
+
+If you encounter migration errors, follow this checklist:
+
+* Ensure that you are running the correct Grakn command (the binary in the Grakn directory of the server you are exporting from or importing to.)
+* Ensure that the schema has been imported correctly to the new keyspace.
+* Ensure that the correct data import path was specified.
+* Ensure that the data was correctly exported by checking the filesize.
+* Ensure that any changed labels are remapped in the import options.
+
+If you have any further errors, please join one of our communities and ask for assistance.
+
+### Incompatible Schema
+
+Between versions, some schemas will become incompatible due to syntax change. Whilst most issues can be corrected in the schema before importing it, it is possible for a label to become invalid (if the label becomes a keyword in a new version). In order to handle this scenario, we have added an option to import from 1.8 onwards that allows you to remap labels during the import.
+
+```
+grakn server import <keyspace> <file>.grakn <old_label>=<new_label>...
+```
+
+### Implicit Relations
+
+Schemas that use implicit relations in 1.7 and earlier (e.g. `@has-attribute`) are not supported by migration tools since they were removed in 1.8. It is therefore recommended that you convert any usage of implicit relations to real relations before migrating to 1.8.
+
+## About `.grakn` Files
```

Comment:
I'd favor not exposing any guidance on modifying the exported files right now -- @lolski what do you think?

Reply:
Have cut it back a bit

Reply:
cool